app/views/vat/moss/eu/registerInHomeCountry.scala.html

```diff
@@ -28,11 +28,12 @@
     <p>@messages("registerInHomeCountry.p1")</p>
     <p>
         @messages("registerInHomeCountry.p2")
-        <a id="call-us"
+        <a id="vat_helpline"
            href='@appConfig.getGovUKUrl("vatEnquiries")'
-           data-journey-click="VatMossEu:Click:CallUs">
+           data-journey-click="VatMossEu:Click:Contact VAT helpline">
```

Comment:
I think we want to keep the GA tags the same. You might want to double-check with Richard Craig

Reply:
Good point. I have reverted GA back to original as ticket doesn't say to change it.